src/components/JoinGroupForm/JoinGroupForm.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useState, useRef } from 'react';
+
+import { InputField } from 'components/InputField/InputField';
+import { Button } from 'components/Button/Button';
+import Modal from 'components/Modal/Modal';
+
+import styles from './JoinGroupForm.module.css';
+
+type ClassInfo = {
+  class_id: string;
+  class_name: string;
+  teacher: string;
+  days: string;
+  time: string;
+};
+
+const dummyInfo = <T,>(): Promise<T> =>
+  new Promise((resolve) => {
+    setTimeout(() => {
+      resolve({
+        class_id: 'ITC023',
+        class_name: 'Tecnologias II',
+        teacher: 'Claudia Perez',
+        days: 'LU-JU',
+        time: '09:00-11:00',
+      } as T);
+    }, 1000);
+  });
+
+export default function JoinGroupForm() {
+  const [isSubmitDisabled, setIsSubmitDisabled] = useState(true);
+  const [classCode, setClassCode] = useState('');
+  const [inputError, setInputError] = useState('');
+  const [classInfo, setClassInfo] = useState<ClassInfo | null>(null);
+  const [isFormSubmitted, setIsFormSubmitted] = useState(false);
+
+  const classCodeRef = useRef<HTMLInputElement>(null);
+  const submitRef = useRef<HTMLButtonElement>(null);
+
+  const checkClasscode = () => {
+    const splittedString = classCode.split(' ');
+    if (splittedString.length > 1) {
+      setInputError('Sin espacios');
```

Comment:
Remember we are handling error messages in utils, so that testing is easier.

Reply:
Thanks! I just moved them to utils